### PR TITLE
Refactor $fixtures property to a getter method

### PR DESCRIPTION
**Pull request by bancer**
https://github.com/cakephp/cakephp/issues/13583

### CODE REVIEW

**Review by ADmad — DISMISSED** [review]

File: src/TestSuite/TestCase.php [hunk 1]
```diff
@@ -831,4 +831,14 @@ public static function setAppNamespace(string $appNamespace = 'TestApp'): void
     {
         Configure::write('App.namespace', $appNamespace);
     }
+
+    /**
+     * Gets fixtures.
+     *
+     * @return array
+     */
+    public function getFixtures()
```
Comment (ADmad) [inline src/TestSuite/TestCase.php]: `array` return type hint required.

**Comment by dereuromark** [comment]
Shouldnt we make $fixtures protected then?

File: src/TestSuite/TestCase.php (hunk 1 above)
Comment (markstory) [inline src/TestSuite/TestCase.php]:
```suggestion
    public function getFixtures(): array
```

File: src/TestSuite/TestCase.php [hunk 2]
```diff
@@ -831,4 +831,14 @@ public static function setAppNamespace(string $appNamespace = 'TestApp'): void
     {
         Configure::write('App.namespace', $appNamespace);
     }
+
+    /**
+     * Gets fixtures.
+     *
+     * @return array
```
Comment (dereuromark) [inline src/TestSuite/TestCase.php]: Isn't this `string[]` ?

Comment (ADmad) [reply src/TestSuite/TestCase.php]:
```suggestion
@return string[]
```

**Comment by dereuromark** [comment]
public $fixtures = [];

I still think we should then make it protected in the process, right?

**Comment by bancer** [comment]
@dereuromark Should it be renamed to $_fixtures?

**Comment by ADmad** [comment]
> Should it be renamed to $_fixtures?

No.

**Pull request merged**